### DISCUSSION TITLE
errors on launcher opening

### DISCUSSION THREAD

**Discussion by francescoameglio** [playbenny/benny#446] [Q&A] [answered]
hello, 

I get these two errors when opening the launcher. Not sure if it's affecting the normal functioning of the software.
![image](https://github.com/user-attachments/assets/d1478d8d-696e-4aa9-bd89-785e6ff78bf2)

cheers!

**Comment by jamesholdenmusic** [comment, marked as answer]
hi francesco,

thanks for reporting!

the first one is a mistake I made that's since been fixed (I shouldn't have put audio_blocks/amxd.Lode.json into the git repository, so you can safely delete that file and the error will go away)

the second is to do with issue #378 - I was trying a particular airwindows reverb to use as a master freeze reverb, but I'm pretty sure I'll use something else (was waiting for the next release of the airwindows externals to come back to this one). i've just pushed an update including that ar.infinity2~ external which should make the error go away.

j

**Reply by francescoameglio** [reply]
thanks James, 

I've got latest and the ar.infinity2 error is gone. 
I could't find any lode.json into the audio_blocks folder. I've actually tried to look for that file throughout the whole repository but no matches.

**Reply by jamesholdenmusic** [reply]
oh! - thanks for checking. mistake wasn't what i thought it was - but i found it referenced in the vst manager patch - have removed and pushed another update which should fix it?

**Reply by francescoameglio** [reply]
Hey James, 
got latest but the error is still there... I wonder if it's referenced somewhere else, maybe in the vstconfiguration?
![image](https://github.com/user-attachments/assets/9a5f235e-7a45-4c48-adbe-4f2e597e66da)

**Reply by jamesholdenmusic** [reply]
ah found one more reference to it, one more update pushed. thanks again for checking!

**Reply by francescoameglio** [reply]
yes, it's now gone, cheers :) 

I'm having a blast with Benny, thanks a lot for sharing it with the rest of the world!
